metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -1098,6 +1118,12 @@ def loop_profiler_requests(
                 ),
             )
 
+    def add_profile_metadata(self, inspector: Inspector) -> None:
```

Comment:
Can we rename it to be more descriptive of what this does?
Like `add_additional_profile_metadata`  or `calculate_table_storage_size` ?
If I got it right, will the GE provide the profile metadata and this only provides the table storage size?